packages/shared/src/table/composable-table.tsx

```diff
@@ -67,6 +71,7 @@ export const ComposableTable: ComposableTableProps = <
         translate={null}
         aria-label="Composable table"
         className="pf-v5-u-mt-md"
+        variant={isCompact ? TableVariant.compact : null}
```

Comment:
totally optional comment:
```suggestion
        {...(isCompact ? {variant: ableVariant.compact} : {})}
```